src/urbit/lib/chess.hoon

```diff
@@ -700,38 +761,62 @@
             ==
         ==
     ==
+  ::
+  ::  apply move if it is a legal move
   ++  apply-move
     |=  move=chess-move
     ^-  (unit chess-position)
     ?.  (legal-move move)  ~
     `(naive-apply move)
+  ::
+  ::  apply a move without checking if it is legal
   ++  naive-apply
     |_  move=chess-move
     ++  $
       ^-  chess-position
       ?-  -.move
+        ::
+        ::  return position at end of game
```

Comment:
Please remove this comment and the remaining comments introduced by this PR for this file. They don't add any information that isn't trivially apparent from inspecting the code.